src/java_bytecode/java_bytecode_instrument.cpp

```diff
@@ -73,29 +69,26 @@ class java_bytecode_instrumentt:public messaget
     const exprt &length,
     const source_locationt &original_loc);
 
-  void instrument_code(exprt &expr);
+  void instrument_code(codet &code);
   void add_expr_instrumentation(code_blockt &block, const exprt &expr);
   void prepend_instrumentation(codet &code, code_blockt &instrumentation);
-  codet instrument_expr(const exprt &expr);
+  optionalt<codet> instrument_expr(const exprt &expr);
 };
 
-
-const std::vector<irep_idt> exception_needed_classes =
-{
+const std::vector<std::string> exception_needed_classes = { // NOLINT
   "java.lang.ArithmeticException",
   "java.lang.ArrayIndexOutOfBoundsException",
   "java.lang.ClassCastException",
   "java.lang.NegativeArraySizeException",
-  "java.lang.NullPointerException"
-};
+  "java.lang.NullPointerException"};
```

Comment:
The existing brace style seems more consistent with our usual style, but I guess clang-format wouldn't take it?

Reply:
clang-format puts it that way, but it's one of those cases where clang-format and cpp-lint disagree